internal/pkg/cli/app.go

```diff
@@ -37,3 +37,22 @@ An application represents an Amazon ECS service or task.`,
 	}
 	return cmd
 }
+
+// BuildDeployCmd is the deploy command - which is
+// an alias for app deploy.
+func BuildDeployCmd() *cobra.Command {
+	deployCmd := BuildAppDeployCmd()
+	deployCmd.Use = "deploy"
```

Comment:
should we add anything to `deployCmd.Aliases`? https://godoc.org/github.com/spf13/cobra#Command

Reply:
I can't think of any good ones. except maybe `yolo` :P